docs/r2/hyperparameter_tuning_with_hparams.ipynb

```diff
@@ -227,113 +206,74 @@
         "2. Dropout rate in the dropout layer\n",
         "3. Optimizer\n",
         "\n",
-        "List the values to try:"
+        "List the values to try, and log an experiment configuration to TensorBoard. This step is optional: it enables you to customize how the hyperparameters and metrics appear in the UI, and which metrics to include."
```

Comment:
Minor: is it about customizing how they appear in the UI (which implies multiple options) or about providing additional detail so the UI can filter more precisely?

Reply:
It’s both. The display name just customizes the UI; the domains provide
additional detail for more precise filtering. I was going for concision,
but I like your phrasing, and it’s probably wise to avoid conveying the
impression that this is _only_ for cosmetic changes. Will fix—thanks!